ext/Pod-Html/lib/Pod/Html.pm

```diff
@@ -384,36 +384,36 @@ sub refine_globals {
 }
 
 sub generate_cache {
-    my ($globals, $Pagesref) = @_;
+    my ($self, $Pagesref) = @_;
     my $pwd = getcwd();
-    chdir($globals->{Podroot}) ||
-        die "$0: error changing to directory $globals->{Podroot}: $!\n";
+    chdir($self->{Podroot}) ||
+        die "$0: error changing to directory $self->{Podroot}: $!\n";
 
     # find all pod modules/pages in podpath, store in %Pages
     # - inc(0): do not prepend directories in @INC to search list;
-    #     limit search to those in @{$globals->{Podpath}}
+    #     limit search to those in @{$self->{Podpath}}
     # - verbose: report (via 'warn') what search is doing
     # - laborious: to allow '.' in dirnames (e.g., /usr/share/perl/5.14.1)
     # - callback: used to remove Podroot and extension from each file
     # - recurse: go into subdirectories
     # - survey: search for POD files in PodPath
     my ($name2path, $path2name) = 
-        Pod::Simple::Search->new->inc(0)->verbose($globals->{Verbose})->laborious(1)
-        ->callback(\&_save_page)->recurse($globals->{Recurse})->survey(@{$globals->{Podpath}});
-    #print STDERR Data::Dumper::Dumper($name2path, $path2name) if ($globals->{Verbose});
+        Pod::Simple::Search->new->inc(0)->verbose($self->{Verbose})->laborious(1)
```

Comment:
Thanks to serendipity for letting me see that there's a boolean property called `laborious` somewhere.  I got a chuckle.

Reply:
> Thanks to serendipity for letting me see that there's a boolean property called `laborious` somewhere. I got a chuckle.

The Pod::Simple stuff was the most laborious part of this whole project.